packages/manager/src/features/Databases/utilities.ts

```diff
@@ -213,6 +216,35 @@ export const toDatabaseFork = (
   return fork;
 };
 
-export const getDatabasesDescription = (database: DatabaseInstance) => {
+export const databaseEngineMap: Record<Engine, string> = {
+  mongodb: 'MongoDB',
+  mysql: 'MySQL',
+  postgresql: 'PostgreSQL',
+  redis: 'Redis',
+};
+
+export const getDatabasesDescription = (
+  database: Pick<DatabaseInstance, 'engine' | 'version'>
+) => {
   return `${databaseEngineMap[database.engine]} v${database.version}`;
 };
+
+export const hasPendingUpdates = (pendingUpdates?: PendingUpdates[]) =>
+  pendingUpdates?.some((update) => update.deadline || update.planned_for);
+
+export const isDefaultDatabase = (
+  database: Pick<DatabaseInstance, 'platform'>
+) => database.platform === 'rdbms-default';
+
+export const isLegacyDatabase = (
+  database: Pick<DatabaseInstance, 'platform'>
+) => !database.platform || database.platform === 'rdbms-legacy';
+
+export const upgradableVersions = (
+  engine: Engine,
+  version: string,
+  engines?: Pick<DatabaseEngine, 'engine' | 'version'>[]
+) =>
+  engines
+    ?.filter((e) => e.engine === engine)
+    ?.filter((e) => e.version > version);
```

Comment:
Can we combine this into single filter instead of filtering twice ? 
```suggestion
engines?.filter((e) => e.engine === engine && e.version > version);

```